addons/web/static/src/core/signature/name_and_signature.js

```diff
@@ -245,7 +245,14 @@ export class NameAndSignature extends Component {
      */
     async printImage(imgSrc) {
         this.clear();
-        await this.fromDataURL(imgSrc);
+        // Force the width and height to the canvas size to prevent shrinking on 4K display
+        // with devicePixelRatio > 1
+        const options = {
+            width: this.signatureRef.el.width,
+            height: this.signatureRef.el.height,
+            yOffset: 30,
```

Comment:
Not sure I see the point of this offset...

Reply:
Indeed, it is not directly related to the issue. It is used to improve the layout. Signature is too high otherwise
![image](https://github.com/odoo/odoo/assets/50832879/08a95b62-bba2-4d9b-8998-5e57e47515c6)